compiler/il/OMRNode.cpp

```diff
@@ -536,6 +536,7 @@ OMR::Node::recreateWithSymRef(TR::Node *originalNode, TR::ILOpCodes op, TR::Symb
 TR::Node *
 OMR::Node::recreateAndCopyValidPropertiesImpl(TR::Node *originalNode, TR::ILOpCodes op, TR::SymbolReference *newSymRef)
    {
+   TR_ASSERT_FATAL(TR::Node::isNotDeprecatedUnsigned(op), "Trying to use a deprecated unsigned opcode: #%d \n", op);
```

Comment:
If these are deprecated, why aren't we removing the opcodes altogether? If the IL opcodes don't exist then we cannot be trying to use a deprecated unsigned opcode at all and hence these asserts and the new function (which is likely to go stale) are not needed.

Reply:
As some deprecated opcodes are still in used in both OMR and OpenJ9, we perform several steps to remove them safely ( https://github.com/eclipse/omr/issues/2657#issuecomment-534270516). We need to make sure these opcodes are no longer generated before we can remove their definition, evaluators, pattern matching, etc.